csharp30thsep/Animals2/Dogs2.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Animals2
+{
+    public class Dogs2:Animals2
+    {
+        
+        public override string sound 
```

Comment:
datamember name should be in pascal case

Reply:
ok